test/es6/toStringTag.js

```diff
@@ -308,7 +230,7 @@ var tests = [
 
             assert.areEqual("[object before\0after]", o.toString(), "ToString implementation handles embedded null characters in @@toStringTag property value");
         }
-    },
+    }

```

Comment:
nit: keeping the trailing comma lets future additions not have an extra line of diff